lib/assets/csv_script.rb

```diff
@@ -0,0 +1,64 @@
+require 'roo'
+require 'csv'
+
+XLSX_PATH = ARGV[0] || "lib/assets/initial-list.xlsx"
+CSV_PATH = ARGV[1] || "lib/assets/pre-chunk-content.csv"
+NEW_CSV_ROOT = ARGV[2] || "lib/assets/2023-01-12-pilot-SNAP-warning-notice_"
+
+xlsx = Roo::Excelx.new(XLSX_PATH)
+
+# Write the data to a new CSV file (no BOM by default)
+xlsx.to_csv(filename=CSV_PATH)
+
+# Optional: explicitly remove BOM if opening a csv that was saved from excel
+# spreadsheet = Roo::Spreadsheet.open("my-csv.csv", { csv_options: { encoding: 'bom|utf-8' } })
+
+header_renames = {
+  "EDG_NUM" => "case_id",
+  "HH Client ID" => "client_id",
+  "HH First Name" => "first_name",
+  "HH Phone Number" => "phone_number",
+  "HH Primary Langauage" => "primary_language",
+}
+csv_data = CSV.read(
+  CSV_PATH,
+  :headers => true,
+  :header_converters => lambda { |header| header_renames[header] || header } # Rename columns
+)
+# Add column preferred_language using column 6 ("HH Primary Langauage")
+csv_data.each do |row|
+  row["preferred_language"] = row[6] == "SPA" ? "es" : "en"
```

Comment:
Here and below, we should be able to remove the hard coding by using `row["primary_language"]` instead of `row[6]`.